shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/HttpUtils.java

```diff
@@ -220,15 +221,16 @@ public String request(final String url, final Map<String, ?> form, final Map<Str
      * @param url    url
      * @param json   json
      * @param header header
+     * @param method method
      * @return Response
      * @throws IOException IOException
      */
     public Response requestJson(final String url, final String json,
-        final Map<String, String> header) throws IOException {
+        final Map<String, String> header, HTTPMethod method) throws IOException {
```

Comment:
where is call this funcation？

Reply:
[HttpUtils](https://github.com/apache/shenyu/pull/5852#discussion_r1893368809)#requestJson,This method is currently invoked in only one place.
The call stack of this method is as follows and the source code location..
org.apache.shenyu.admin.controller.SandboxController#proxyGateway. Line 52 - 55
    org.apache.shenyu.admin.service.impl.SandboxServiceImpl#requestProxyGateway. Line 79 - 110
        org.apache.shenyu.admin.utils.HttpUtils#requestCall. Line 303 - 308
            org.apache.shenyu.admin.utils.HttpUtils#requestJson. Line 226 - 231

Reply:
You changed the method signature, don't you need to change it in the call?

Reply:
Hi，pls check ci，it not passed

Reply:
Applied Checkstyle to HttpUtils file